plugins/slick.cellrangeselector.js

```diff
@@ -71,13 +71,13 @@
       _isBottomCanvas = _$activeCanvas.hasClass('grid-canvas-bottom');
 
       if (_gridOptions.frozenRow > -1 && _isBottomCanvas) {
-        _rowOffset = (_gridOptions.frozenBottom) ? $('.grid-canvas-bottom').height() : $('.grid-canvas-top').height();
+        _rowOffset = (_gridOptions.frozenBottom) ? $(`.${_grid.getUID()} .grid-canvas-bottom`).height() : $(`.${_grid.getUID()} .grid-canvas-top`).height();
```

Comment:
could you please change the syntax to the old ES5 syntax of concatenating text + variable. This new ES6 syntax you have here won't work in legacy browser and we want to keep it as ES5 compatible at the source

Reply:
@ghiscoding, I have done the changes as per your request.